### PR TITLE
Added template_fields_renderers for MySQL Operator

### PR DESCRIPTION
**Pull request by oyarushe**
Added template_fields_renderers for MySQL Operator, it relates to issue #11177 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: airflow/providers/mysql/operators/mysql.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ class MySqlOperator(BaseOperator):
     """
 
     template_fields = ('sql',)
+    template_fields_renderers = {'sql': 'sql'}
```
Comment (potiuk) [inline airflow/providers/mysql/operators/mysql.py]: Nice, but we need a unit test here.

Comment (turbaszek) [reply airflow/providers/mysql/operators/mysql.py:49]: Should we add `json` renderer for `parameters`?

File: airflow/providers/mysql/operators/mysql.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ class MySqlOperator(BaseOperator):
...
     template_ext = ('.sql',)
     ui_color = '#ededed'
```
Comment (turbaszek) [inline airflow/providers/mysql/operators/mysql.py:52]: Should we add `json` renderer for `parameters`?

Comment (potiuk) [reply airflow/providers/mysql/operators/mysql.py:52]: That's a good idea. Lets' do it!

File: airflow/providers/mysql/operators/mysql.py (hunk 1 above)
Comment (turbaszek) [reply airflow/providers/mysql/operators/mysql.py]: Interesting point, I don't think we have test for `template_fields_renderes` yet. What we probably should check is that renderers are proper values:
https://github.com/apache/airflow/blob/01fb7775ddcf8fdaa9afbfd60d66cc616a8f5226/airflow/www/utils.py#L351-L370

Comment (potiuk) [reply airflow/providers/mysql/operators/mysql.py]: Right. That works for me. Can we see the screenshot ?

**Review by potiuk — CHANGES_REQUESTED** [review]
Screenshot and parameters rendering would be nice!

File: airflow/providers/mysql/operators/mysql.py (hunk 1 above)
Comment (oyarushe) [reply airflow/providers/mysql/operators/mysql.py]: Please, check the screenshots:
image
Test DAG:
image
Rendered Template:
image

**Review by potiuk — APPROVED** [review]

**Review by turbaszek — APPROVED** [review]
Nice one!

**Pull request merged**